File: Unidad 3/Java/Programa_1.java


package com.mycompany.programa_1;
import java.util.Scanner;
/**
 *
 * @author LaboratorioLTI2
 */
public class Programa_1 {

    public static void main(String[] args) {
        int edad;
        Scanner sc = new Scanner(System.in);
        System.out.print("Ingrese su edad: ");
        edad=sc.nextInt();
        
        if(edad<18){
             System.out.println("Usted es menor de edad");
        }else{
             System.out.println("Usted es mayor de edad");
        }
}}

File: Unidad 3/C#/Programa_1.cs
using System;
namespace Progrma_1;

class Edad
{
    static void Main(string[] args)
    {
        int edad;
        Console.Write("¿Cuál es su edad? ");
        edad=Convert.ToInt32(Console.ReadLine());

        if (edad < 18)
        {
            Console.WriteLine("Es menor de edad");
        }
        else
        {
            Console.WriteLine("Es mayor de edad");
        }
    }
}